dbus-daemon-launch-helper: fix build with libxml

dbus-daemon-launch-helper links with the trival config parser, which
doesn't provide bus_config_parser_check_doctype(), like the big parser does.
The libxml loader unfortunately wants to call it, so this breaks the build.

Fix it by including a dummy bus_config_parser_check_doctype() in the trivial
config parser.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 bus/config-parser-trivial.c |    8 ++++++++
 1 file changed, 8 insertions(+)

Index: dbus-1.2.12/bus/config-parser-trivial.c
===================================================================
--- dbus-1.2.12.orig/bus/config-parser-trivial.c
+++ dbus-1.2.12/bus/config-parser-trivial.c
@@ -310,6 +310,14 @@
   return &parser->service_dirs;
 }
 
+dbus_bool_t
+bus_config_parser_check_doctype (BusConfigParser   *parser,
+                                 const char        *doctype,
+                                 DBusError         *error)
+{
+    return TRUE;
+}
+
 #ifdef DBUS_BUILD_TESTS
 #include <stdio.h>
 #include "test.h"
